enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/InteractiveQuestion.tsx

```diff
@@ -65,7 +65,7 @@ export const _InteractiveQuestion = ({
   const hasQuestionChanges =
     card && (!card.id || card.id !== card.original_card_id);
 
-  const [loading, setLoading] = useState(true);
+  const [isQuestionLoading, setIsQuestionLoading] = useState(true);
```

Comment:
Renaming to make is obvious this means it's loading a question *not* running a question query